samples/apps/copilot-chat-app/SKWebApi/Skills/ChatSkill.cs

```diff
@@ -302,7 +302,7 @@ public async Task SaveNewMessageAsync(string message, SKContext context)
     {
         var allMemories = this.GetAllMemoriesAsync(context);
 
-        return await allMemories.FirstAsync();
+        return await allMemories.FirstOrDefaultAsync();
```

Comment:
This change fixes the case when no memories are found (for example, when there is no memory store)